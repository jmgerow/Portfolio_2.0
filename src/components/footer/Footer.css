#footer-icons {
    color: white;
    margin-left: 6px;
    margin-right: 6px;
}