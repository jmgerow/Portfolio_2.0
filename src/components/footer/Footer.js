import React, { Component } from 'react';
import { Link } from 'react-router-dom';

import './Footer.css';

class Footer extends Component {
    constructor() {
        super();

        this.state = {
            copy: "&copy 2019"
        }
    }

    render() {
        return (
            <footer className="page-footer blue-grey">
                <div className="footer-copyright">
                    <div className="container">
                    &copy; 2019
                        <b className="blue-grey-text text-lighten-3"> Justin Gerow  </b>
                        <a href="https://github.com/jmgerow" target="_blank" rel="noopener noreferrer"><i className="fab fa-github fa-2x" id="footer-icons"></i></a>
                        <a href="https://www.linkedin.com/in/justingerow/" target="_blank" rel="noopener noreferrer"><i className="fab fa-linkedin fa-2x" id="footer-icons"></i></a>
                        <a href="https://twitter.com/jmgerow" target="_blank" rel="noopener noreferrer"><i className="fab fa-twitter-square fa-2x" id="footer-icons"></i></a>
                    </div>
                </div>
            </footer>
        );
    }
}

export default Footer;