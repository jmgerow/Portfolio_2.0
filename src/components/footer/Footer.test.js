import React from 'react';
import { shallow } from 'enzyme';
import Footer from './Footer'

describe('Footer', () => {
    const footer = shallow(<Footer />)
  
    it('renders without crashing', () => {
      expect(footer).toMatchSnapshot();
    });
 
    // it('displays name', () => {
    //   expect(navbar.state().title).toEqual("Justin Gerow")
    // });
  });
