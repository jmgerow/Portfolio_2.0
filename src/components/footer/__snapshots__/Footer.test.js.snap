// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders without crashing 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Footer />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="footer-copyright"
      >
        <div
          className="container"
        >
          © 2019
          <b
            className="blue-grey-text text-lighten-3"
          >
             Justin Gerow  
          </b>
          <a
            href="https://github.com/jmgerow"
            rel="noopener noreferrer"
            target="_blank"
          >
            <i
              className="fab fa-github fa-2x"
              id="footer-icons"
            />
          </a>
          <a
            href="https://www.linkedin.com/in/justingerow/"
            rel="noopener noreferrer"
            target="_blank"
          >
            <i
              className="fab fa-linkedin fa-2x"
              id="footer-icons"
            />
          </a>
          <a
            href="https://twitter.com/jmgerow"
            rel="noopener noreferrer"
            target="_blank"
          >
            <i
              className="fab fa-twitter-square fa-2x"
              id="footer-icons"
            />
          </a>
        </div>
      </div>,
      "className": "page-footer blue-grey",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="container"
        >
          © 2019
          <b
            className="blue-grey-text text-lighten-3"
          >
             Justin Gerow  
          </b>
          <a
            href="https://github.com/jmgerow"
            rel="noopener noreferrer"
            target="_blank"
          >
            <i
              className="fab fa-github fa-2x"
              id="footer-icons"
            />
          </a>
          <a
            href="https://www.linkedin.com/in/justingerow/"
            rel="noopener noreferrer"
            target="_blank"
          >
            <i
              className="fab fa-linkedin fa-2x"
              id="footer-icons"
            />
          </a>
          <a
            href="https://twitter.com/jmgerow"
            rel="noopener noreferrer"
            target="_blank"
          >
            <i
              className="fab fa-twitter-square fa-2x"
              id="footer-icons"
            />
          </a>
        </div>,
        "className": "footer-copyright",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "© 2019",
            <b
              className="blue-grey-text text-lighten-3"
            >
               Justin Gerow  
            </b>,
            <a
              href="https://github.com/jmgerow"
              rel="noopener noreferrer"
              target="_blank"
            >
              <i
                className="fab fa-github fa-2x"
                id="footer-icons"
              />
            </a>,
            <a
              href="https://www.linkedin.com/in/justingerow/"
              rel="noopener noreferrer"
              target="_blank"
            >
              <i
                className="fab fa-linkedin fa-2x"
                id="footer-icons"
              />
            </a>,
            <a
              href="https://twitter.com/jmgerow"
              rel="noopener noreferrer"
              target="_blank"
            >
              <i
                className="fab fa-twitter-square fa-2x"
                id="footer-icons"
              />
            </a>,
          ],
          "className": "container",
        },
        "ref": null,
        "rendered": Array [
          "© 2019",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": " Justin Gerow  ",
              "className": "blue-grey-text text-lighten-3",
            },
            "ref": null,
            "rendered": " Justin Gerow  ",
            "type": "b",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <i
                className="fab fa-github fa-2x"
                id="footer-icons"
              />,
              "href": "https://github.com/jmgerow",
              "rel": "noopener noreferrer",
              "target": "_blank",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "fab fa-github fa-2x",
                "id": "footer-icons",
              },
              "ref": null,
              "rendered": null,
              "type": "i",
            },
            "type": "a",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <i
                className="fab fa-linkedin fa-2x"
                id="footer-icons"
              />,
              "href": "https://www.linkedin.com/in/justingerow/",
              "rel": "noopener noreferrer",
              "target": "_blank",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "fab fa-linkedin fa-2x",
                "id": "footer-icons",
              },
              "ref": null,
              "rendered": null,
              "type": "i",
            },
            "type": "a",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <i
                className="fab fa-twitter-square fa-2x"
                id="footer-icons"
              />,
              "href": "https://twitter.com/jmgerow",
              "rel": "noopener noreferrer",
              "target": "_blank",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "fab fa-twitter-square fa-2x",
                "id": "footer-icons",
              },
              "ref": null,
              "rendered": null,
              "type": "i",
            },
            "type": "a",
          },
        ],
        "type": "div",
      },
      "type": "div",
    },
    "type": "footer",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="footer-copyright"
        >
          <div
            className="container"
          >
            © 2019
            <b
              className="blue-grey-text text-lighten-3"
            >
               Justin Gerow  
            </b>
            <a
              href="https://github.com/jmgerow"
              rel="noopener noreferrer"
              target="_blank"
            >
              <i
                className="fab fa-github fa-2x"
                id="footer-icons"
              />
            </a>
            <a
              href="https://www.linkedin.com/in/justingerow/"
              rel="noopener noreferrer"
              target="_blank"
            >
              <i
                className="fab fa-linkedin fa-2x"
                id="footer-icons"
              />
            </a>
            <a
              href="https://twitter.com/jmgerow"
              rel="noopener noreferrer"
              target="_blank"
            >
              <i
                className="fab fa-twitter-square fa-2x"
                id="footer-icons"
              />
            </a>
          </div>
        </div>,
        "className": "page-footer blue-grey",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="container"
          >
            © 2019
            <b
              className="blue-grey-text text-lighten-3"
            >
               Justin Gerow  
            </b>
            <a
              href="https://github.com/jmgerow"
              rel="noopener noreferrer"
              target="_blank"
            >
              <i
                className="fab fa-github fa-2x"
                id="footer-icons"
              />
            </a>
            <a
              href="https://www.linkedin.com/in/justingerow/"
              rel="noopener noreferrer"
              target="_blank"
            >
              <i
                className="fab fa-linkedin fa-2x"
                id="footer-icons"
              />
            </a>
            <a
              href="https://twitter.com/jmgerow"
              rel="noopener noreferrer"
              target="_blank"
            >
              <i
                className="fab fa-twitter-square fa-2x"
                id="footer-icons"
              />
            </a>
          </div>,
          "className": "footer-copyright",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "© 2019",
              <b
                className="blue-grey-text text-lighten-3"
              >
                 Justin Gerow  
              </b>,
              <a
                href="https://github.com/jmgerow"
                rel="noopener noreferrer"
                target="_blank"
              >
                <i
                  className="fab fa-github fa-2x"
                  id="footer-icons"
                />
              </a>,
              <a
                href="https://www.linkedin.com/in/justingerow/"
                rel="noopener noreferrer"
                target="_blank"
              >
                <i
                  className="fab fa-linkedin fa-2x"
                  id="footer-icons"
                />
              </a>,
              <a
                href="https://twitter.com/jmgerow"
                rel="noopener noreferrer"
                target="_blank"
              >
                <i
                  className="fab fa-twitter-square fa-2x"
                  id="footer-icons"
                />
              </a>,
            ],
            "className": "container",
          },
          "ref": null,
          "rendered": Array [
            "© 2019",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": " Justin Gerow  ",
                "className": "blue-grey-text text-lighten-3",
              },
              "ref": null,
              "rendered": " Justin Gerow  ",
              "type": "b",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <i
                  className="fab fa-github fa-2x"
                  id="footer-icons"
                />,
                "href": "https://github.com/jmgerow",
                "rel": "noopener noreferrer",
                "target": "_blank",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "className": "fab fa-github fa-2x",
                  "id": "footer-icons",
                },
                "ref": null,
                "rendered": null,
                "type": "i",
              },
              "type": "a",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <i
                  className="fab fa-linkedin fa-2x"
                  id="footer-icons"
                />,
                "href": "https://www.linkedin.com/in/justingerow/",
                "rel": "noopener noreferrer",
                "target": "_blank",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "className": "fab fa-linkedin fa-2x",
                  "id": "footer-icons",
                },
                "ref": null,
                "rendered": null,
                "type": "i",
              },
              "type": "a",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <i
                  className="fab fa-twitter-square fa-2x"
                  id="footer-icons"
                />,
                "href": "https://twitter.com/jmgerow",
                "rel": "noopener noreferrer",
                "target": "_blank",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "className": "fab fa-twitter-square fa-2x",
                  "id": "footer-icons",
                },
                "ref": null,
                "rendered": null,
                "type": "i",
              },
              "type": "a",
            },
          ],
          "type": "div",
        },
        "type": "div",
      },
      "type": "footer",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
